results/user-testing-results.md

```diff
@@ -0,0 +1,138 @@
+# Results: JupyterLab Accessibility Overview
+
+These results are from user interviews conducted in May 2023 with [this test script]() on [a modified version of NumPy Tutorials’ Saving and sharing your NumPy arrays notebook](https://github.com/Quansight-Labs/JupyterLab-user-testing/blob/main/notebooks/save-load-arrays.ipynb). 
+
+This work was funded by a [Chan Zuckerberg Initiative Essential Open Source Software for Science grant](https://chanzuckerberg.com/eoss/proposals/inclusive-and-accessible-scientific-computing-in-the-jupyter-ecosystem/). The [full grant proposal can be found on the jupyter/accessibility repository](https://github.com/jupyter/accessibility/blob/da6db43da092255c205ce1603d0d10b1f51c131b/docs/funding/Inclusive_and_Accessible_Scientific_Computing_in_Jupyter_Ecosystem_SUBMITTED_PROPOSAL.pdf).
```

Comment:
Would it make sense to move this paragraph to the bottom? in a new Acknowledgements section?

I'm also wondering if it would make sense to insert a very brief overview of the testing script and interview: 

> Participants were first asked to complete a set of tasks: to find information in a notebook, change settings in the JupyterLab UI, modify cells in a notebook, etc. Then they were interviewed about their experience completing the tasks and about JupyterLab in general. 

something like that?

Reply:
+1 on moving this to the bottom, usually funding sources tend to go there

Reply:
Deleting funding notes here and adding @gabalafou's description.

```suggestion
Participants were first asked to complete a set of tasks: to open files in JupyterLab, find information in a notebook, modify cells in a notebook, change settings in the JupyterLab UI, etc. Then they were interviewed about their experience completing the tasks and about JupyterLab in general.
```